packages/fxa-settings/src/components/HeaderLockup/index.test.tsx

```diff
@@ -5,14 +5,20 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
 import '@testing-library/jest-dom/extend-expect';
+import { MockedProvider } from '@apollo/client/testing';
+import { createCache } from '../../models/_mocks';
 import HeaderLockup from '.';
 
 // TO DO: functional test for `data-testid="header-menu"` to be visible in
 // mobile & tablet but hidden at desktop
 
 describe('HeaderLockup', () => {
   it('renders as expected', () => {
-    render(<HeaderLockup avatarUrl={null} primaryEmail="user@example.com" />);
+    render(
+      <MockedProvider cache={createCache({ account: { avatarUrl: '' } })}>
```

Comment:
If the `avatarUrl` isn't set, we get back `null` and not an empty string

Reply:
oh yeah, that's me mid debugging something else ;)